src/test/java/edu/ucsb/cs156/happiercows/controllers/CommonsControllerTests.java

```diff
@@ -152,8 +153,9 @@ public void deleteCommonsThatDoesNotExistTest() throws Exception {
       .andExpect(status().isNotFound()).andReturn();
 
     verify(commonsRepository, times(1)).findById(1L);
-    String responseString = response.getResponse().getContentAsString();
-    assertEquals("{\"type\":\"EntityNotFoundException\",\"message\":\"Commons with id 1 not found\"}", responseString);
+    Map<String, Object> json = responseToJson(response);
```

Comment:
Fixed the issue by mapping the response to a json and testing the `type` and `message` parts of the expected response, so the test will pass regardless of the output's order.

Reply:
This issue came up when @jasonem225 was writing tests for the edit endpoint as well, he did the same and now the edit tests are working properly.